### PR TITLE
chore: make vote count int32 and some tests changes

### LINKED ISSUE #11: Changes related to pillar block format from C++
- vote count change is int32 in C++, currently int96 is used in solidity - should be int32
```
  struct WeightChange {
        address validator;
        int96 change;
    }
```
- optional: struct WeightChange represents vote count change for each validator. Could be therefore renamed to VoteCountChange
- all pillar block checks for period+1 are not valid, pillar blocks periods are actual pbft periods when they were created and we will create pillar block each N pbft periods - this number is in node config. We could either get rid of this check or somehow get this N from config ? Could be potentially also hardcoded but we might use different values for different networks
```
require(finalized.block.period + 1 == pb.block.period, "Block number + 1 != finalized block number");
 require(pending.block.period + 1 == pb.block.period, "Block period + 1 != pending block period");
```
might be on some other places too ?

### PR DESCRIPTION
**Pull request by kstdl**
Fixes #11

### CODE REVIEW

File: src/connectors/ERC20LockingConnector.sol [hunk 1]
```diff
@@ -10,9 +10,9 @@ import "./TokenConnectorBase.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
 contract ERC20LockingConnector is TokenConnectorBase {
-    constructor(address bridge, IERC20 token, address tara_addresss_on_eth)
+    constructor(address bridge, IERC20 token, address other_network_address)
```
Comment (VargaElod23) [inline src/connectors/ERC20LockingConnector.sol:13]: I guess `token_on_other_network` if we wanna be more precise

File: src/connectors/ERC20MintingConnector.sol [hunk 2]
```diff
@@ -8,9 +8,9 @@ import "./TokenConnectorBase.sol";
 import "./IERC20MintableBurnable.sol";
 
 contract ERC20MintingConnector is TokenConnectorBase {
-    constructor(address bridge, IERC20MintableBurnable token, address tara_addresss_on_eth)
+    constructor(address bridge, IERC20MintableBurnable token, address other_network_address)
```
Comment (VargaElod23) [inline src/connectors/ERC20MintingConnector.sol:11]: Same

File: test/Fees.t.sol [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import "../src/lib/Constants.sol";
+
+contract FeesTest is Test {
+    BridgeLightClientMock taraLightClient;
+    BridgeLightClientMock ethLightClient;
+    TestERC20 ethTaraToken;
+    TaraBridge taraBridge;
+    EthBridge ethBridge;
+
+    address caller = address(bytes20(sha256(hex"1234")));
```
Comment (VargaElod23) [inline test/Fees.t.sol:21]: You can use vm.addr

**Review by VargaElod23 — APPROVED** [review]
Small comments added but LGTM

File: src/eth/TaraClient.sol [hunk 4]
```diff
@@ -202,8 +203,8 @@ contract TaraClient is IBridgeLightClient {
         internal
     {
         uint256 gasleftbefore = gasleft();
-        require(finalized.block.period + 1 == b.block.period, "Pending block should have number 1 greater than latest");
-        require(b.block.prevHash == finalized.blockHash, "Pending block must be child of latest");
+        require(finalized.block.period + 1 == b.block.period, "block.number != finalized.number +1");
```
Comment (JakubFornadel) [inline src/eth/TaraClient.sol:206]: period + 1 ?

Comment (kstdl) [reply src/eth/TaraClient.sol:206]: Yeah, removed it everywhere, but for some reason forgot to do it here. Fixed

**Review by JakubFornadel — APPROVED** [review]

**Pull request merged**